api/openapi-spec/swagger.json

```diff
@@ -2,7 +2,7 @@
   "swagger": "2.0",
   "info": {
    "title": "Kubernetes",
-   "version": "v1.6.0"
+   "version": "v1.6.1"
```

Comment:
The Jenkins verification on this PR failed due to `hack/verify-federation-openapi-spec.sh`, so I ran `./hack/update-openapi-spec.sh` and amended the commit. I'm not sure if the verify failure would have sorted itself out without any intervention, but I can pull that change back out so it's purely a cherry-pick of the master commit.

This is my first cherry-pick, so apologies if I've bungled the process.